project/src/main/resources/ScapegoatInspections.scala

```diff
@@ -42,6 +42,6 @@ private[scapegoat] final case class ScapegoatInspection(
   defaultLevel: Level
 )
 
-private[scapegoat] object ScapegoatInspections {
+object ScapegoatInspections {
```

Comment:
Why made it _public_ ?
With the work you did in #130 this can be _private to the `scapegoat` **package**, the same can go with the `ScalastyleInspections` **object**.

Reply:
Looks like I missed it after refactoring.